### ISSUE TITLE
[Question] What are the possibilities of having images and tooltips with tksheet.

### ISSUE THREAD

**Issue by Pjuske** [ragardner/tksheet#47]
First of all, thanks for this amazing tool 👍 

Is there an easy solution to having an image rather than text in a cell?
Is there an easy solution to have a tooltip binding on a cell?

Thanks in advance

**Comment by ragardner** [comment]
Thank you for your interest in the library,

There is currently not an easy solution to put an image in a cell, I think, sorry. One of the issues is the way redrawing currently works if the cell isn't big enough to fit the whole image then it wouldn't show it at all, as far as I'm aware the solution to this is drawing background color boxes in every cell (which would be a highlight box if not background color) onto which the text goes, the advantage of this is there would not longer be any need to calculate text widths and lines and text lines could by half rendered

tldr: it's an overhaul that interests me and I might add the functionality in the future


On the tool tip functionality... there isn't a built in method to do it so you'd have to write one yourself, possibly by binding mouse motion to your own function but let me know if you write anything for the library

**Comment by ragardner** [comment]
@Pjuske Unfortunately after trying to implement the half text rendering I found it was not easily possible and slowed down redrawing significantly due to the extra rectangles being moved around

I will not be adding images to tksheet anytime soon sorry :(

**Comment by Pjuske** [comment]
> @Pjuske Unfortunately after trying to implement the half text rendering I found it was not easily possible and slowed down redrawing significantly due to the extra rectangles being moved around
> 
> I will not be adding images to tksheet anytime soon sorry :(

Thanks for the follow-up!
